draft-ietf-dots-requirements.md

```diff
@@ -158,28 +158,28 @@ Mitigator:
   detected or reported DDoS attack. For the purposes of this document, this
   entity is a black box capable of mitigation, making no assumptions about
   availability or design of countermeasures, nor about the programmable
-  interface between this entity and other network elements. The mitigator and
-  DOTS server are assumed to belong to the same administrative entity.
+  interface(s) between this entity and other network elements. The mitigator and
+  invoked DOTS server are assumed to belong to the same administrative entity.
 
 DOTS client:
 : A DOTS-aware software module responsible for requesting attack response
   coordination with other DOTS-aware elements.
 
 DOTS server:
 : A DOTS-aware software module handling and responding to messages from DOTS
-  clients. The DOTS server SHOULD enable mitigation on behalf of the DOTS
+  clients. The DOTS server enables mitigation on behalf of the DOTS
   client, if requested, by communicating the DOTS client's request to the
   mitigator and returning selected mitigator feedback to the requesting DOTS
-  client. A DOTS server MAY also be a mitigator.
+  client. A DOTS server may also be colocated with a mitigator.
 
 DOTS agent:
 : Any DOTS-aware software module capable of participating in a DOTS signal
-  or data channel.
+  or data channel. It can be a DOTS client, DOTS server, or DOTS gateway.
```

Comment:
A DOTS gateway itself is explicitly not a DOTS agent.

Reply:
Good point about the description of gateways. Maybe this? `... It can be a DOTS client, a DOTS server, or, as a logical DOTS agent, a DOTS gateway.`